programmable_video/android/src/main/kotlin/twilio/flutter/twilio_programmable_video/TwilioProgrammableVideoPlugin.kt

```diff
@@ -70,6 +70,17 @@ class TwilioProgrammableVideoPlugin : FlutterPlugin {
 
         lateinit var pluginHandler: PluginHandler
 
+        @JvmStatic
```

Comment:
I thought it was fine to remove this?

Reply:
Left it in there for when the release flag is off.

Reply:
Was this an issue with camera1/2? I thought it was a WebRTC issue?

Reply:
Yes webrtc issue, but in case that this causes any issues, its release flagged.